platform/lib/pipeline/markdownDocument.js

```diff
@@ -157,13 +171,21 @@ class MarkdownDocument {
   }
 
 
-  _convertSyntax() {
-    this._contents = MarkdownDocument.rewriteCalloutToTip(this._contents);
-    this._contents = MarkdownDocument.rewriteCodeBlocks(this._contents);
-    this._contents = MarkdownDocument.escapeMustacheTags(this._contents);
-
-    // Replace dividers (---) as they will break front matter
-    this._contents = this._contents.replace(/\n---\n/gm, '\n***\n');
+  /**
+   * Will add the markdown="1" attribute for table tags in order to
+   * let the markdown.extensions.extra of grow evaluate markdown inside the table.
+   * Needed because often code blocks are inside the table.
+   * @param  {String} contents
+   * @return {String} The rewritten input
+   */
+  static enableMarkdownInHtmlTables(contents) {
+    return contents.replace(TABLE_MARKDOWN_ATTRIB_PATTERN, (match, p1, p2) => {
+      // only if we are not in a code block set the markdown attribute
+      if (p1) {
+        return p1 + ' markdown="1"' + p2;
```

Comment:
Is `markdown="1"` going to be removed by the markdown extension? Otherwise, this would lead to invalid AMPHTML.

Reply:
Jep, the attribute is removed by python-markdown.